src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/FeedbackReviews/hooks/useFeedbackReviews.test.tsx

```diff
@@ -0,0 +1,260 @@
+        id: 'e6c03f46-d6de-4a3c-8456-c87fdf6d8650',
+        name: 'ItemA-123',
+        order: 1,
+        question: { en: 'A-123' },
+        responseType: 'multiSelect',
+        responseValues: {
+          options: [
+            {
+              id: 'd79a89a2-d6ce-44db-8ff7-3033b1826514',
+              isNoneAbove: false,
+              text: 'A123',
+              value: 0,
+            },
+            {
+              id: '9cb16928-b20e-4c8d-84d5-131c44cd2c90',
+              isNoneAbove: false,
+              text: 'A456',
+              value: 1,
+            },
+            {
+              id: 'f33efef3-bffb-4880-9a96-b7f287dd6533',
+              isNoneAbove: true,
+              text: 'None',
+              value: 2,
+            },
+          ],
+          type: 'multiSelect',
+        },
+      },
+      answer: undefined,
+      items: undefined,
+    },
+  ];
+
+  beforeEach(() => {
+    (useFormContext as jest.Mock).mockReturnValue(mockUseFormContext);
+    (useFeedbackReviewsData as jest.Mock).mockReturnValue({
+      fetchReviewsData: mockFetchReviewsData,
+    });
+
+    (useAsync as jest.Mock).mockImplementation((apiFunction, onSuccess) => ({
+      execute: jest.fn(async (params) => {
+        const result = await apiFunction(params);
+        await onSuccess(result);
+      }),
+      isLoading: false,
+      error: null,
+    }));
+  });
+
+  test('should fetch reviews correctly', async () => {
+    const mockReviewsResult = { data: { result: [{ id: 1, answer: 'encrypted answer' }] } };
+    (getReviewsApi as jest.Mock).mockResolvedValue(mockReviewsResult);
+    mockFetchReviewsData.mockResolvedValue([{ reviewId: 1, reviewer: 'test' }]);
+
+    const { result } = renderHook(
+      () =>
+        useFeedbackReviews({
+          appletId: 'testAppletId',
+          answerId: 'testAnswerId',
+          submitId: null,
+          user: { id: 'testUserId', firstName: 'Jane', lastName: 'Doe' },
+          setAssessmentStep: jest.fn(),
+        }),
+      {
+        wrapper: ({ children }) => (
+          <RespondentDataReviewContext.Provider value={mockContextValue}>
+            {children}
+          </RespondentDataReviewContext.Provider>
+        ),
+      },
+    );
+
+    await act(async () => {
+      await result.current.handleGetReviews();
+    });
+
+    expect(getReviewsApi).toHaveBeenCalledWith({
+      appletId: 'testAppletId',
+      answerId: 'testAnswerId',
+    });
+
+    expect(mockFetchReviewsData).toHaveBeenCalledWith({
+      reviews: [{ id: 1, answer: 'encrypted answer' }],
+      userId: 'testUserId',
+    });
+
+    expect(result.current.reviewerData).toEqual([{ reviewId: 1, reviewer: 'test' }]);
+  });
+
+  test('should handle review removal correctly (activity)', async () => {
+    (deleteReviewApi as jest.Mock).mockResolvedValue({});
+    (getReviewsApi as jest.Mock).mockResolvedValue({ data: { result: [] } });
+    mockFetchReviewsData.mockResolvedValue([]);
+
+    const { result } = renderHook(
```

Comment:
It seems you can move the code from line 185 to line 201 into its own function (e.g., renderUseFeeedbackReviews) and provide parameters that can be changed. This way, it can be reused in other places.